pages/glossary/display-size.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Display size
+key: display-size
+unambiguous: true
+objective: true
+---
+
+The display size is the width and height at which a page is rendered. The display size is equal to the [innerWidth][] and [innerHeight][] of the [window][] of the [top-level browsing context][].
```

Comment:
If referring to inner width & height of the window, perhaps using the keyword, viewport size would be a better terminology.

Reply:
That's a good point. The term "display" can really mean many things; viewport, monitor, the `display` property. The term "viewport", however, has fewer meanings.

Reply:
I'll rename to viewport. Fair point.